docs/source/tutorials/e2e_flow.rst

```diff
@@ -164,78 +227,68 @@ modifying its associated config ``eleuther_evaluation.yaml``.
     to install the EleutherAI evaluation harness.
 
 Since we plan to update all of the checkpoint files to point to our fine-tuned checkpoints,
-let's first copy over the config to our local working directory so we can make changes. This
-will be easier than overriding all of these elements through the CLI.
+let's first copy over the config to our local working directory so we can make changes.
```

Comment:
Actually I'd consider scrapping the custom eval config altogether. The reason we had it before was because it was too many nasty overrides to do neatly from the CLI. Now that there are only two fields to override, it should be much nicer (just make sure to override the lists correctly).

If you do decide to keep the custom config I would at least truncate it somewhat. E.g. you don't need to put everything below the "OTHER PARAMETERS" line explicitly

Reply:
Here it’s also a bit nasty, we have to update the model type and tokenizer path, so it’s 4 args, but not thar bad